Core/WKWebViewConfigurationExtension.swift

```diff
@@ -101,6 +101,19 @@ private class Loader {
         } else {
             load(scripts: [ .messaging, .apbfilterES2015 ], forMainFrameOnly: false)
         }
+        
+        if isDebugBuild {
+            let signpostMessaging = "try { webkit.messageHandlers.signpostMessage.postMessage(data); } catch(error) {}"
```

Comment:
I'm not keen on having JS in strings in our swift code. How about inserting an alternative JS depending on the isDebugBuild flag instead?

Reply:
Done, along with 'log' message handler.